.github/workflows/config-e2e.yml

```diff
@@ -0,0 +1,42 @@
+name: Firebase Remote Config Rollouts E2E Tests
+
+on:
+  schedule:
+    - cron: 24 */4 * * *  # every 4 hours @ 24 minutes past the hour
```

Comment:
Any reason for 24 minutes?

Reply:
Not really, just using the same frequency that Sessions used. This can be updated as we see fit in the future.